include/sound/sof/info.h

```diff
@@ -25,6 +25,7 @@
 #define SOF_IPC_INFO_LOCKS		BIT(1)
 #define SOF_IPC_INFO_LOCKSV		BIT(2)
 #define SOF_IPC_INFO_GDB		BIT(3)
+#define SOF_IPC_INFO_IMR_RESTORE	BIT(4)
 
```

Comment:
@dbaluta Do you see any need for a fast-boot flag indiciated by FW? 

This could be perhaps SOF_IPC_INFO_FASTBOOT

... if defined, FW supports retaining some state even if if suspended. The actual details would be vendor specific, but we could use a common flag in FW_READY that all vendors could use.

Reply:
and the second line is a bit confusing right? you're not showing how to choose the feature or not just yet. So its better to just say: "If the bit is set, the driver can skip downloading the firmware again during system resume or runtime resume"